tests/test_extract_zone_tops.py

```diff
@@ -0,0 +1,54 @@
+import os
+
+import pytest
+
+from fmu.tools.qcforward.extract_grid_zone_tops import extract_grid_zone_tops
+
+
+if "__file__" in globals():
+    # Easen up copying test code into interactive sessions
+    testdir = os.path.dirname(os.path.abspath(__file__))
+else:
+    testdir = os.path.abspath(".")
+
+
+GRID = os.path.join(testdir, "data/reek_sim_grid.roff")
+GRIDPROP = os.path.join(testdir, "data/reek_sim_zone.roff")
```

Comment:
Rather use xtgeo testrepo directly, as ../xtgeo-testdata/... so we avoid filling this repo with large chunks of tesdata